android/src/main/java/io/ionic/portals/reactnative/ReactNativePortalsPubSub.kt

```diff
@@ -24,7 +24,7 @@ internal class PortalsPubSubModule(reactContext: ReactApplicationContext) :
         val ref = PortalsPubSub.shared.subscribe(topic) { result ->
             reactApplicationContext
                 .getJSModule(DeviceEventManagerModule.RCTDeviceEventEmitter::class.java)
-                .emit("PortalsSubscription:$eventName", result.toJSObject().toReactMap())
```

Comment:
The tl;dr is that this turned into "PortalsSubscription:PortalsSubscription:TheEventName"